packages/console/src/containers/ConsoleContent/index.tsx

```diff
@@ -155,13 +155,15 @@ function ConsoleContent() {
               </Route>
             </Route>
             {isDevFeaturesEnabled && (
-              <Route path="organizations">
-                <Route index element={<Organizations />} />
-                <Route path="create" element={<Organizations />} />
-                <Route path="settings" element={<Organizations tab="settings" />} />
-                <Route path=":id/*" element={<OrganizationDetails />} />
-                <Route path="guide" element={<OrganizationGuide />} />
-              </Route>
+              <>
+                <Route path="organizations">
+                  <Route index element={<Organizations />} />
+                  <Route path="create" element={<Organizations />} />
+                  <Route path="settings" element={<Organizations tab="settings" />} />
+                  <Route path=":id/*" element={<OrganizationDetails />} />
+                </Route>
+                <Route path="organization-guide/*" element={<OrganizationGuide />} />
```

Comment:
nit: `create-organization/*`

Reply:
The last step in the guide is actually called `create-organization`, so I'm just trying to avoid having `/create-organization/create-organization` route name... But maybe we can rename the last step. Will double check with the designer today